### PR TITLE
Add document for tiup cluster clean

### PR DESCRIPTION
**Pull request by lucklove**
Signed-off-by: lucklove <gnu.crazier@gmail.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: maintain-tidb-using-tiup.md [hunk 1]
```diff
@@ -197,6 +197,21 @@ tiup cluster stop ${cluster-name} -R tidb
 tiup cluster stop ${cluster-name} -N 1.2.3.4:4000,1.2.3.5:4000
 ```
 
+## 清除集群数据
+
+此操作会关闭所有服务，并清空其数据目录或/和日志目录，并且无法恢复，需要**谨慎操作**。
```
Comment (TomShawn) [inline maintain-tidb-using-tiup.md:202]: `和/或`是什么意思呢？

File: maintain-tidb-using-tiup.md [hunk 2]
```diff
@@ -197,6 +197,21 @@ tiup cluster stop ${cluster-name} -R tidb
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster clean ${cluster-name} --data                                   # 清空集群所有服务的数据，但保留日志
+tiup cluster clean ${cluster-name} --log                                    # 清空集群所有服务的日志，但保留数据
+tiup cluster clean ${cluster-name} --all                                    # 清空集群所有服务的数据和日志
+tiup cluster clean ${cluster-name} --all --ignore-role prometheus           # 清空 prometheus 以外的所有服务的日志和数据
+tiup cluster clean ${cluster-name} --all --ignore-node 172.16.13.11:9000    # 清空节点 172.16.13.11:9000 以外的所有服务的日志和数据
+tiup cluster clean ${cluster-name} --all --ignore-node 172.16.13.12         # 清空部署在 172.16.13.12 以外的所有服务的日志和数据
```
Comment (TomShawn) [inline maintain-tidb-using-tiup.md:212]: @lucklove 可否按照上面的格式将命令与注释拆开呢？

清空集群所有服务的数据，但保留日志：

{{< copyable "shell-regular" >}}

```bash
tiup cluster clean ${cluster-name} --data
```

清空集群所有服务的日志，但保留数据：

{{< copyable "shell-regular" >}}

```bash
tiup cluster clean ${cluster-name} --log
```

Comment (lonng) [reply maintain-tidb-using-tiup.md:212]: 非常好的建议

File: maintain-tidb-using-tiup.md [hunk 3]
```diff
@@ -197,6 +197,56 @@ tiup cluster stop ${cluster-name} -R tidb
...
+
+清空节点 172.16.13.11:9000 以外的所有服务的日志和数据：
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster clean ${cluster-name} --all --ignore-node 172.16.13.11:9000
+```
+
+清空部署在 172.16.13.12 以外的所有服务的日志和数据：
```
Comment (TomShawn) [inline maintain-tidb-using-tiup.md:242]:
```suggestion
清空部署在 `172.16.13.12` 以外的所有服务的日志和数据：
```

File: maintain-tidb-using-tiup.md [hunk 4]
```diff
@@ -197,6 +197,56 @@ tiup cluster stop ${cluster-name} -R tidb
...
+
+清空 prometheus 以外的所有服务的日志和数据：
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster clean ${cluster-name} --all --ignore-role prometheus
+```
+
+清空节点 172.16.13.11:9000 以外的所有服务的日志和数据：
```
Comment (TomShawn) [inline maintain-tidb-using-tiup.md:234]:
```suggestion
清空节点 `172.16.13.11:9000` 以外的所有服务的日志和数据：
```

File: maintain-tidb-using-tiup.md [hunk 5]
```diff
@@ -197,6 +197,56 @@ tiup cluster stop ${cluster-name} -R tidb
...
+
+清空集群所有服务的数据和日志：
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster clean ${cluster-name} --all 
+```
+
+清空 prometheus 以外的所有服务的日志和数据：
```
Comment (TomShawn) [inline maintain-tidb-using-tiup.md:226]:
```suggestion
清空 Prometheus 以外的所有服务的日志和数据：
```

**Pull request merged**